### ISSUE TITLE
where is moving_average update ?

### ISSUE THREAD

**Issue by Bingohong** [titu1994/Keras-Group-Normalization#2]
I found that there is no moving_average_mean/var, which is necessary in BatchNorm step. the group normalization without moving_average updat , it is oK?

**Comment by titu1994** [comment]
There is no moving average for group Normalization.
